src/main/kotlin/com/openlattice/assembler/processors/EntitySetContainsFlagEntryProcessor.kt

```diff
@@ -0,0 +1,14 @@
+package com.openlattice.assembler.processors
+
+import com.openlattice.edm.EntitySet
+import com.openlattice.edm.set.EntitySetFlag
+import com.openlattice.rhizome.hazelcast.entryprocessors.AbstractReadOnlyRhizomeEntryProcessor
+import java.util.*
+
+data class EntitySetContainsFlagEntryProcessor(val flag: EntitySetFlag)
```

Comment:
Any reason why this isn't also Offloadable?

Reply:
it is now!

Reply:
......except apparently the push failed. Now it is actually